frontend/sozisel-app/src/components/LoginScreen/LoginScreen.tsx

```diff
@@ -0,0 +1,90 @@
+import "./LoginScreen.scss";
+
+import * as Yup from "yup";
+
+import Button from "../utils/Button/Button";
+import Card from "../utils/Card/Card";
+import ErrorMessage from "../utils/Input/ErrorMessage";
+import Input from "../utils/Input/Input";
+import Navbar from "../Navbar/Navbar";
+import { ReactElement } from "react";
+import conference_img from "../../assets/conference_img.png";
+import { useForm } from "react-hook-form";
+import { useHistory } from "react-router";
+import { useLoginMutation } from "../../graphql";
+import { useTranslation } from "react-i18next";
+import { yupResolver } from "@hookform/resolvers/yup";
+
+interface LoginFormData {
+  email: string;
+  password: string;
+}
+
+export default function LoginScreen(): ReactElement {
+  const { t } = useTranslation("common");
+
+  const loginSchema = Yup.object().shape({
```

Comment:
I cannot do that because I use `t` variable (`useTranslation`  hook) to define error messages in `loginSchema`, and the react hooks cannot be called at the top level, outside the component function.

Reply:
You can pass translation string as error messages and inside function pass that string to `t` variable. No need to use `t` before displaying.

Reply:
You can pass translation string as error messages and inside function pass that string to `t` variable. No need to use `t` before displaying.